docs/community/com-monitoring/com-monitoring-datadog.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Setup Datadog monitoring
+layout: default
+parent: Community monitoring
+grand_parent: Community
+nav_order: 1
+---
+
+# How do I setup Datadog monitorning?
+
+You can configure FeatureBase Community to output Prometheus metrics which can be read by Datadog.
+
+{: .note}
+[Learn how to setup Datadog to read EC2 Cloudwatch metrics](https://docs.datadoghq.com/integrations/amazon_web_services/?tab=roledelegation#installation){:target="\_blank"} for FeatureBase systems installed on AWS EC2.
+
+## Before you begin
+
+* [Learn about FeatureBase monitoring](/docs/community/com-monitoring/com-monitoring-home)
+* [Install Python3 if not already installed](https://www.python.org/downloads/){:target="_blank"}
```

Comment:
Any reason this is hear? I don't see it mentioned in the old page

Reply:
Because on the Datadog agent readme it says it needs python 7

https://github.com/DataDog/datadog-agent

Reply:
Ya but they will see that there. Perhaps they change this in their README in the future, now we are out of date and we don't know it. Let's let them keep that information